.github/workflows/mocha.yml

```diff
@@ -0,0 +1,157 @@
+      - name: find commit msg for PR
+        id: commitMsg
+        if: github.event_name == 'pull_request'
+        run: >-
+          echo "::set-output name=HEAD_COMMIT_MSG::$(git log --no-merges -1
+          --oneline)"
+  check-skip:
+    name: Check to skip CI
+    needs: prepare-commit-msg
+    runs-on: ubuntu-latest
+    if: >-
+      ${{ !contains(github.event.head_commit.message, '[ci skip]') &&
+      !contains(needs.prepare-commit-msg.outputs.HEAD_COMMIT_MSG, '[ci skip]')
+      }}
+    steps:
+      - run: 'echo "${{ github.event.head_commit.message }}"'
+  smoke:
+    name: 'Smoke Test [Node.js v${{ matrix.node }} / ${{ matrix.os }}]'
+    needs: check-skip
+    runs-on: '${{ matrix.os }}'
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - windows-2019
+        node:
+          - 10
+          - 12
+          - 14
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '${{ matrix.node }}'
+      - run: npm install --production
+      - run: npm run test:smoke
+  test-node:
+    name: 'Test Node.js [v${{ matrix.node }} / ${{ matrix.os }}]'
+    needs: smoke
+    runs-on: '${{ matrix.os }}'
+    env:
+      NODE_OPTIONS: '--trace-warnings'
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - windows-2019
+        node:
+          - 10
+          - 12
+          - 14
+        include:
+          - os: ubuntu-latest
+            node: 14
+            env:
+              COVERAGE: 1
+    steps:
+      - name: Cache Growl Installer (Windows)
+        if: "${{ matrix.os == 'windows-2019' }}"
+        id: cache-growl
+        uses: actions/cache@v2
+        with:
+          path: GrowlInstaller
+          key: '${{ runner.os }}-growl-installer'
+          restore-keys: |
+            ${{ runner.os }}-growl-installer
+      - name: Download Growl Installer (Windows)
+        if: "${{ matrix.os == 'windows-2019' && steps.cache-growl.outputs.cache-hit != 'true'}}"
+        run: >
+          echo "Downloading Growl installer..."
+
+          mkdir GrowlInstaller | out-null
+
+          $seaURL =
+          "https://github.com/briandunnington/growl-for-windows/releases/download/final/GrowlInstaller.exe"
+
+          $seaPath = "GrowlInstaller\GrowlInstaller.exe"
+
+          $webclient = New-Object Net.WebClient
+
+          $webclient.DownloadFile($seaURL, $seaPath)
+
+          7z x $seaPath -oGrowlInstaller | out-null
+
+      - name: Retrieve Growl Installer (Windows)
+        if: "${{ matrix.os == 'windows-2019' }}"
+        uses: actions/cache@v2
+        with:
+          path: GrowlInstaller
+          key: '${{ runner.os }}-growl-installer'
+          restore-keys: |
+            ${{ runner.os }}-growl-installer
+      - name: Add Growl Installer to Path (Windows)
+        if: "${{ matrix.os == 'windows-2019' }}"
+        run: echo "C:\Program Files (x86)\Growl for Windows" >> $GITHUB_PATH
```

Comment:
Nice. [I just found the deprecating announcement](https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/).

Reply:
I was unable to get this working.  [Going to try this suggestion now](https://github.community/t/migration-from-deprecated-add-path-on-windows/136265)